devicetypes/smartthings/zwave-multi-metering-switch.src/zwave-multi-metering-switch.groovy

```diff
@@ -30,6 +30,12 @@ metadata {
 		fingerprint mfr: "0000", cc: "0x5E,0x25,0x27,0x32,0x81,0x71,0x60,0x8E,0x2C,0x2B,0x70,0x86,0x72,0x73,0x85,0x59,0x98,0x7A,0x5A", ccOut:"0x82", ui:"0x8700", deviceJoinName: "Aeotec Switch 1" //Aeotec Nano Switch 1
 		fingerprint mfr: "027A", prod: "A000", model: "A004", deviceJoinName: "Zooz Switch" //Zooz ZEN Power Strip
 		fingerprint mfr: "027A", prod: "A000", model: "A003", deviceJoinName: "Zooz Switch" //Zooz Double Plug
+		fingerprint mfr: "015F", prod: "3102", model: "0201", deviceJoinName: "WYFY Switch 1", mnmn: "SmartThings", vid: "generic-switch" //WYFY Touch 1-button Switch
```

Comment:
Please, could You add a Raw Description in a comment above each of Your new fingerprints ?
(it can be found at graph.api.smartthings.com -> My devices -> [select device] -> Raw Description ...)

so it would look like:

```
// Raw Description...
fingerprint...
// Raw Description...
fingerprint...
// Raw Description...
fingerprint...
etc.
```

Reply:
@KKlimczukS Added as requested.